app/notify_client/__init__.py

```diff
@@ -54,16 +56,26 @@ def check_inactive_service(self):
         ):
             abort(403)
 
+    def check_inactive_user(self):
+        if os.getenv("NOTIFY_E2E_TEST_EMAIL"):
```

Comment:
Let's leave a `TODO` comment here to remind ourselves that this is another spot for E2E config cleanup for the future (e.g., using a feature flag for it):

```suggestion
        # TODO:  Update this once E2E tests are managed by a feature flag or some other main config option.
        if os.getenv("NOTIFY_E2E_TEST_EMAIL"):
```